iothub_client/tests/common_e2e/iothubclient_common_e2e.c

```diff
@@ -949,14 +1214,51 @@ static void send_event_test(IOTHUB_PROVISIONED_DEVICE* deviceToUse, IOTHUB_CLIEN
         // cleanup
         destroy_d2c_message_handle(d2cMessage);
     }
+}
+
+static void send_batch_event_test(IOTHUB_PROVISIONED_DEVICE* deviceToUse, IOTHUB_CLIENT_TRANSPORT_PROVIDER protocol)
+{
+    TEST_MESSAGE_CREATION_MECHANISM test_message_creation[] = { TEST_MESSAGE_CREATE_BYTE_ARRAY, TEST_MESSAGE_CREATE_STRING };
+
+    int i;
+    for (i = 0; i < sizeof(test_message_creation) / sizeof(test_message_creation[0]); i++)
+    {
+        // arrange
+        SEND_DATA_BUNDLE d2cMessages = { 0 };
+
+        clear_connection_status_info_flags();
+
+        // Create the IoT Hub Data
+        client_ll_connect_to_hub(deviceToUse, protocol);
+
+        // Send the Events from the client
+        client_create_and_send_d2c_messages(test_message_creation[i], 5, LARGE_MESSAGE_SIZE, &d2cMessages);
```

Comment:
`5` is hard-coded magic constant should be declared at top of file.

Reply:
Defined LARGE_MESSAGE_COUNT